lib/types.js

```diff
@@ -570,6 +570,13 @@ function getBuilderName(typeName) {
         }
     });
 }
+exports.getBuilderName = getBuilderName;
+
+function getStatementBuilderName(typeName) {
+    typeName = getBuilderName(typeName);
+    return typeName.replace(/(Expression)?$/, "Statement");
+}
+exports.getStatementBuilderName = getStatementBuilderName;
```

Comment:
I expose this to make testing easier. I also exposed `getBuilderName` for consistency. Neither is not exposed in `main` however, so the public API does not really change.